source/frontend/src/features/leases/detail/LeasePages/payment/table/periods/PaymentPeriodsView.tsx

```diff
@@ -121,8 +122,12 @@ export const PeriodPaymentsView: React.FunctionComponent<
           icons: { open: <MdArrowDropDown size={24} />, closed: <MdArrowRight size={24} /> },
         }}
       />
-    </Section>
+    </StyledPayments>
   );
 };
 
+const StyledPayments = styled(Section)`
+  min-width: 130rem;
```

Comment:
The payments screen isn't really functional below this size - so if we shrink to this extent just set a hard minimum that will require scrolling.